Tests/SystemTests/PublisherSubscriberSystemTests.swift

```diff
@@ -3,7 +3,7 @@ import AblyAssetTrackingCore
 import AblyAssetTrackingPublisher
 import AblyAssetTrackingSubscriber
 
-class PublisherSubscriber: XCTestCase {
```

Comment:
The intention of this test class is for the publisher to publish and the subscriber to subscribe to those location updates. This would be the same as the Android library.

Reply:
ead77e36171b7a31fd3d07a9077325eee2fe69b7